src/components/FreeResponseInput.tsx

```diff
@@ -148,16 +148,16 @@ export const FreeResponseInput = (props: FreeResponseProps) => {
         <div className="controls">
           <RevertButton disabled={!textHasChanged} onClick={cancelHandler} />
           <SubmitBtn
-              data-test-id="submit-answer-btn"
-              disabled={isSubmitDisabled || isOverWordLimit}
-              onClick={saveHandler}
+            data-test-id="submit-answer-btn"
+            disabled={isSubmitDisabled || isOverWordLimit}
+            onClick={saveHandler}
           >
-              {submitBtnLabel}
+            {submitBtnLabel}
           </SubmitBtn>
         </div>
       </StepCardFooter>
     </StyledFreeResponse>
-);
+  );
```

Comment:
Sorry about all the formatting changes, I forgot I'd turned on a formatter. Though nice to have it all consistent I suppose.